shared/controlflow/codeql/controlflow/BasicBlock.qll

```diff
@@ -64,6 +80,7 @@ module Make<InputSig Input> {
     BasicBlock getAPredecessor(SuccessorType t) { result.getASuccessor(t) = this }
 
     /** Gets the control flow node at a specific (zero-indexed) position in this basic block. */
+    cached
```

Comment:
It needs to be defined inside the `Cached` module, and then just referenced here.

Reply:
> It should be possible to reference them with the `BasicBlocks:: prefix`?

Yes, that works. I didn't think of that 🙈

Reply:
Is it right now? `bbIndex` and friends are in the cached module an `getNode` is uncached.